front/package.json

```diff
@@ -76,6 +76,7 @@
     "react-dnd-touch-backend": "^16.0.1",
     "react-select": "^4.3.1",
     "react-slider": "^2.0.6",
+    "react-tag-input-component": "^2.0.2",
```

Comment:
Why use this library instead of react-select that we already use?

https://react-select.com/

The "multi" option + "creatable" seems to do the same?

<img width="641" alt="Screenshot 2023-09-28 at 10 00 26" src="https://github.com/GladysAssistant/Gladys/assets/7365207/7aad87b4-4a32-4e66-9323-138abb93fc03">

(Maybe the UX of the one you picked is better, but let's see if it's worth having a secondary lib?)